src/System.Security.Cryptography.Xml/tests/Samples/SigningVerifyingX509Cert.cs

```diff
@@ -60,6 +60,13 @@ private static bool VerifyXml(string signedXmlText, X509Certificate2 certificate
         }
 
         [Fact]
+        [SkipOnTargetFramework(
+            TargetFrameworkMonikers.Net45 |
+            TargetFrameworkMonikers.Net451 |
+            TargetFrameworkMonikers.Net452 |
+            TargetFrameworkMonikers.Net46 |
```

Comment:
Why are net45, net451, net452, and net46 in this list?  We outright don't support them.